test/mailers/developer_mailer_test.rb

```diff
@@ -0,0 +1,15 @@
+require "test_helper"
+
+class DeveloperMailerTest < ActionMailer::TestCase
+  test "welcome developer email test" do
+    @developer = Developer.first
+    email = DeveloperMailer.with(developer: @developer).developer_welcome_email
+
+    assert_emails 1 do
+      email.deliver_now
+    end
+
+    assert_equal [@developer.user.email], email.to
+    assert_not_nil email.body.to_s
```

Comment:
```suggestion
    assert_match /added your developer profile/, email.body.to_s
```

Reply:
Test won't pass unless I use email.body.encoded instead, is this OK?

Reply:
Yeah go for it!